### PR TITLE
Sync CODEOWNERS to governance committeee

### PR DESCRIPTION
**Pull request by lizthegrey**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -12,4 +12,4 @@
 #  https://help.github.com/en/articles/about-code-owners
 #
 
-*  @bhs @bogdandrutu @sarahnovotny @SergeyKanzhelev @yurishkuro
+*  @opentelemetry/admins
```
Comment (yurishkuro) [inline CODEOWNERS]: is the admins list actually visible publicly?

image

Comment (lizthegrey) [reply CODEOWNERS]: We should make it publicly visible IMO.

Comment (lizthegrey) [reply CODEOWNERS]: @yurishkuro you have access to update the visibility settings.

Comment (yurishkuro) [reply CODEOWNERS]: changed. I assume it's not case sensitive.

**Comment by lizthegrey** [comment]
Hm, now the shield icon is gone from everyone. This doesn't seem right :(

**Pull request merged**